docs/dusty-specs/app-specs.md

```diff
@@ -0,0 +1,132 @@
+# Dusty App Specs
+These specs are used to specify apps for you local environment. They should go in the apps folder in your specs repo.
+
+## Keys
+
+### repo
```

Comment:
well hmm. I change my mind to no, because I want to keep repo and mount together and image and build together